app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagServiceEndpointV2.java

```diff
@@ -100,15 +100,44 @@ public ConnectionTestResult testConnection()
 
     public FeedsCredentials getCredentials() throws RequestException, IOException {
         FeedsCredentials fc = getCredentials("/config", CREDENTIALS_PATTERN);
-        // overwrite userID with username because first matcher group of previous regex, which
-        // should return the user name, might include the subdirectory in which wallabag is installed
-        if(fc != null) fc.userID = username;
+        // fc.userID might include the subdirectory in which wallabag is installed.
+        // If feed url is "https://example.com/wallabag/complex/user/name/token/unread.xml",
+        // fc.userID will be "wallabag/complex/user/name", but should be "complex/user/name".
+        // The following code is an attempt to work around this issue.
+        if(fc != null) {
```

Comment:
since we have no else block here, i suggest inverting the condition and directly return if fc is null. This way we can move the whole content of this if block some spaces left and reduce the nesting.